crates/net/discv4/src/node.rs

```diff
@@ -1,22 +1,9 @@
-use crate::{proto::Octets, PeerId};
-use bytes::{Buf, BufMut};
 use generic_array::GenericArray;
-use reth_primitives::keccak256;
-use reth_rlp::{Decodable, DecodeError, Encodable};
-use reth_rlp_derive::RlpEncodable;
-use secp256k1::{SecretKey, SECP256K1};
-use std::{
-    fmt,
-    fmt::Write,
-    net::{IpAddr, Ipv4Addr, SocketAddr},
-    num::ParseIntError,
-    str::FromStr,
-};
-use url::{Host, Url};
+use reth_primitives::{keccak256, NodeRecord, PeerId};
 
 /// The key type for the table.
 #[derive(Debug, Copy, Clone, Eq, PartialEq)]
-pub(crate) struct NodeKey(pub(crate) PeerId);
+pub struct NodeKey(pub(crate) PeerId);
```

Comment:
these don't need to be pub, right?

Reply:
You are right. After moving it to _primitives_ and back, I mistakenly left it public. Thanks for catching it!